packages/core/src/assets/predict.js

```diff
@@ -0,0 +1,41 @@
+const modelDefine = require('./modelDefine').default;
+const fastify = require('fastify')({ logger: true });
```

Comment:
Could we remove the fastify dependence and only export as a library? This makes integration to other frameworks to be easy more :)

Reply:
Yes, i'm thinking we could remove fastify dependency here, and just leave the pure prediction function so that it's easy to intergrate to other systems. But in the meantime, we can add a command in cli such as `pipcook try(predict) ...` to use fastify quickly run a server locally (just for beginners to quickly check results intuitively). Does this make sense

Reply:
How about `pipcook serve`? Because the `try` sub command might be reversed for MLApp.

Reply:
done~